AEPCore/Sources/rules/LaunchRulesEngine.swift

```diff
@@ -293,6 +293,16 @@ public class LaunchRulesEngine {
             case is [String: Any]:
                 let valueDict = mutableDict[key] as! [String: Any]
                 mutableDict[key] = replaceToken(in: valueDict, data: data)
+            case is [Any]:
+                let valueArr = mutableDict[key] as! [Any]
```

Comment:
Can we extend support for nested token replacement in arrays to maintain consistency with maps?

Reply:
Good call. I have added support and made it more generic to handle nested dictionaries inside nested arrays and vice versa.